src/libslic3r/GCode/ToolOrdering.hpp

```diff
@@ -150,11 +150,11 @@ class ToolOrdering
 
     // For the use case when each object is printed separately
     // (print->config().print_sequence == PrintSequence::ByObject is true).
-    ToolOrdering(const PrintObject &object, unsigned int first_extruder, bool prime_multi_material = false);
+    ToolOrdering(const PrintObject &object, unsigned int first_extruder, bool is_BBL_printer, bool prime_multi_material = false);
```

Comment:
same here. 
We can query it by callin object.print().is_BBL_printer()

Reply:
Ah I didn’t notice that, will correct soon. 🙏🏻

Reply:
Done

Reply:
Thank you. 
All look good now 👍